tests/test_lms_api.py

```diff
@@ -126,7 +126,26 @@ def test_get_course_enrollment_not_found():
 
 
 @responses.activate
+def test_get_enrollment_course_modes():
+    course_id = "course-v1:edX+DemoX+Demo_Course"
+    response = {"course_modes": ['a', 'list', 'containing', 'course', 'modes']}
+    expected_return = ['a', 'list', 'containing', 'course', 'modes']
+    responses.add(
+        responses.GET,
+        _url(
+            "enrollment",
+            "course/{}".format(course_id),
+        ),
+        json=response
+    )
+    client = lms_api.EnrollmentApiClient()
+    actual_response = client.get_course_modes(course_id)
+    assert actual_response == expected_return
+
+
+@responses.activate
 def test_get_enrolled_courses():
+    # Copied from https://github.com/edx/edx-enterprise/pull/109
```

Comment:
@itsjeyd Sorry, this comment should go in the previous method, which is the one I copied from https://github.com/edx/edx-enterprise/pull/109/files#diff-82ee99218a64ea6e5fb77cb5c7db8fd3